### PR TITLE
banner

### PR DESCRIPTION
**Pull request by HrikB**

### CODE REVIEW

File: dapp/src/components/layout.js [hunk 1]
```diff
@@ -97,6 +97,18 @@ const Layout = ({
           </>
         )}
       </Head>
+
+      <div
+        className={classnames('notice text-white text-center p-3 dapp d-none')}
+      >
+        <div className="container d-flex flex-column flex-md-row align-items-center justify-content-center">
+          <span className="text">
+            OETH has been paused as a security precaution. Minting and redeeming
```
Comment (shahthepro) [inline dapp/src/components/layout.js:106]: Shouldn't this be OUSD

File: dapp-oeth/src/components/layout.js [hunk 2]
```diff
@@ -68,7 +68,18 @@ const Layout = ({
           content="https://ousd.com/images/share-twitter.png"
         />
       </Head>
-
+      <div
```
Comment (shahthepro) [inline dapp-oeth/src/components/layout.js:71]: Can we put these alerts behind a ENV var so that we don't have to submit a PR and deploy it everytime?

**Review by shahthepro — APPROVED** [review]

File: dapp-oeth/src/components/layout.js (hunk 2 above)
Comment (shahthepro) [reply dapp-oeth/src/components/layout.js:71]: Looking at this again, I guess it makes more sense to read from the contracts directly. This should never require deployment or ENV change

Comment (rafaelugolini) [reply dapp-oeth/src/components/layout.js:71]: I think we are over using ENV variables. The usage of const is also good :)

**Pull request closed without merging**